cli_flags.go

```diff
@@ -67,6 +68,7 @@ var (
 		"Valid values are in the range [1..%d], and 0 to disable off-cpu profiling."+
 		"Default is %d.",
 		support.OffCPUThresholdMax, defaultOffCPUThreshold)
+	envVarsHelp = "Defines a comma seperated list of env vars that should be reported with the captured profiling samples"
```

Comment:
```suggestion
	envVarsHelp = "Comma seperated list of environment variables that should be reported with the captured profiling samples."
```

Reply:
Are wildcards supposed to be supported?

Reply:
Not for our usecase

Reply:
I think with wild cards the mentioned data security issues might become more relevant